valhalla/baldr/tilehierarchy.h

```diff
@@ -53,6 +53,15 @@ class TileHierarchy {
    */
   static GraphId GetGraphId(const midgard::PointLL& pointll, const uint8_t level);
 
+  /**
+   * Returns the grid Id within the tile. A tile is subdivided into a nxn grid.
+   * The grid Id within the tile is used to sort nodes spatially within the tile.
+   * @param pointll  Lat,lng location within the tile.
+   * @param level    Level of the requested tile.
+   * @return  Returns a grid id used for sorting nodes.
+   */
+  static uint32_t GetGridId(const midgard::PointLL& pointll, const uint8_t level);
```

Comment:
can this stay an implementation detail inside of graphbuilder.cc rather than come into the public api (ie in the headers)

Reply:
done